changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1st December 2022
+
+* Added operation [Merge accounts](../operations/accounts.md#merge-accounts).
+* Deprecated operation [Merge customers](../operations/customers.md#merge-customers). Operation [Merge accounts](../operations/accounts.md#merge-accounts) should be used instead.
```

Comment:
I agree, we should add a label `Deprecated` on the operation itself and the entry in the Operations contents page.

Reply:
Do you have any preference or idea how we wanna put it in the operation content page?

Reply:
@MikeAdamsMews ^

Reply:
> Do you have any preference or idea how we wanna put it in the operation content page?

That's a reasonable question. Right now the answer is no, because we've never deprecated an endpoint before so there is no precedent. My first thought is to use strikethrough on the name, like this ~~Merge customers~~ and in the Description simply write `Deprecated`, however I really need to do some research and test some possibilities to find out what works best.

Reply:
@MikeAdamsMews So do we wanna exclude the deprecation from the documentation for now or we leave it like this and on operation itself we leave it as it is.

Reply:
I suggest this for operations/README.md:

| [~~Merge customers~~](customers.md#merge-customers) | **Deprecated!** Please use [Merge accounts](accounts.md#merge-accounts) instead. |
___

I suggest this for customers.md:

## ~~Merge customers~~

> ### Deprecated!
> This operation is [deprecated](../deprecations/README.md), please use [Merge accounts](accounts.md#merge-accounts) instead.

Merges one customer to another. All entities attached to the source customer \(e.g. orders, bills\) are attached to the target customer. Profile information of the target customer are extended but not overwritten with profile information of the source customer.